container/build_tar.py

```diff
@@ -77,6 +82,9 @@
                           'Specify the owner names of individual files, e.g. '
                           'path/to/file=root.root.')
 
+gflags.DEFINE_string(
+    'root_directory', './', 'Default root directory is named "."')
```

Comment:
Windows Docker image layers store the files in a root directory called "Files" instead of one named "." like Linux.

Updated documentation to reflect this also.

Reply:
awesome, this clears things up, will look at the updated docs.